src/react/account/AccountDetails.jsx

```diff
@@ -0,0 +1,78 @@
+// @flow
+import Table, * as T from '../ui-elements/TableKeyValue';
+import type { AppState } from '../../types/state';
+import { Clipboard } from '../ui-elements/Clipboard';
+import { CustomTabs } from '../ui-elements/Tabs';
+import React from 'react';
+import { formatDate } from '../utils';
+import styled from 'styled-components';
+import { useSelector } from 'react-redux';
+
+const Title = styled.div`
+    font-size: 17px;
+`;
+
+const TableContainer = styled.div`
+    width: fit-content;
+    margin: 15px 0 0 30px;
+`;
+
+function AccountDetails() {
+
+    const account = useSelector((state: AppState) => state.account.display);
+
+    if (!account.id) {
+        return null;
+    }
+
+    return (
+        <CustomTabs
+            items={[
+                {
+                    onClick: function noRefCheck(){},
+                    selected: true,
+                    title: 'Properties and keys',
+                },
+            ]}
+        >
+            <Title> Account details </Title>
+            <TableContainer>
+                <Table id='account-details-table'>
+                    <T.Body>
+                        <T.Row>
+                            <T.Key> Account ID </T.Key>
+                            <T.Value> {account.id} </T.Value>
+                            <T.ExtraCell> <Clipboard text={account.id}/> </T.ExtraCell>
+                        </T.Row>
+                        <T.Row>
+                            <T.Key> Name </T.Key>
+                            <T.Value> {account.userName} </T.Value>
+                            <T.ExtraCell> <Clipboard text={account.userName}/> </T.ExtraCell>
+                        </T.Row>
+                        <T.Row>
+                            <T.Key> Creation Date </T.Key>
+                            <T.Value> {formatDate(new Date(account.createDate))} </T.Value>
+                        </T.Row>
+                        <T.Row>
+                            <T.Key> Quota (GB) </T.Key>
+                            <T.Value> {account.quotaMax || 'no'} </T.Value>
```

Comment:
minor preference: instead of `no`, could be `N/A` (not applicable) for example, or `none`